### DISCUSSION TITLE
Como é trabalhar para uma consultoria outsourcing?

### DISCUSSION THREAD

**Discussion by leandrocrs** [frontendbr/forum#1132] [Discussão]
Qualquer um com *angular* ou *react* no currículo mais cedo ou mais tarde será assediado por uma consultoria terceirizada prestando serviços a alguma gigante do ramo financeiro.

No período em que estive a procura, participei de alguns processos seletivos, e conversando com colegas do ramo acabei ficando com uma péssima impressão sobre trabalhar para consultorias. As queixas geralmente são:

* Ambiente estressante - o prazo é apertado e os recursos são subdimensionados, a bomba pode estourar a qualquer momento
* Rotatividade de projetos - você leva tempo para se apegar ao projeto que você cuida, e tão logo é produtivo nele, te realocam para outro lugar
* Abandono no cliente - te alocam no cliente e esquecem que você existe
* Falta de perspectiva de mudanças - essa vem acompanhada da anterior, o que você é hoje é o que você será amanhã
* Cultura workaholic - sair no horário é geralmente mal visto pelos líderes e equipe

Outros tantos relatam que gostam de trabalhar neste formato, jamais foram explorados, e que isso é do profissional que se deixa ser explorado, seja por falta de experiência ou confiança ou qualquer outra razão.

Não raro recebo propostas com salários bem interessantes, mas fico com receio (preconceito talvez ?) e não dou prosseguimento depois que sou selecionado para vagas de alocação.

---
Pessoal do fórum que trabalha que tem experiência com consultorias, é mesmo esse bicho papão?
---

**Para dar tempero a discussão:**
Thread dos nossos irmãos portugueses sobre trabalhar em consultorias Reddit
https://www.reddit.com/r/portugal/comments/3xwq9b/trabalhar_em_consultoria_n%C3%A3o_obrigado/

Porque não trabalhar em consultorias - GUJ
http://www.guj.com.br/t/porque-nao-trabalhar-em-consultorias/19428

Existe vida fora de consultoria? - GUJ
http://www.guj.com.br/t/existe-vida-fora-de-consultoria/18090

**Comment by felipefialho** [comment]
Quem pode ajudar aqui é  @grippado

**Comment by rubenmarcus** [comment]
Opa compartilho a mesma impressão. Fora os processos seletivos desgastantes, a oferta salarial é atraente mesmo, mas me falaram algumas pessoas que trabalham em algumas famosas, que tem um esquema que quem não tem empresa PJ, a propria empresa fornece uma empresa em que vc entra como sócio, e eles tem uma contabilidade deles lá que cobra ja um valor de nota fixo e dos serviços também.

Algumas fazem o esquema PJ com beneficios CLT,mas não são todas.
Alguns conhecidos falam que entrar numa empresa dessas ,é esquecer da vida, feriados, fins de semana e etc, você vai trabalhar.

**Comment by kelciocajueiro** [comment]
Eu trabalhei praticamente minha carreira toda assim. Todos os pontos elencados pelo @leandrw são a pura verdade.

Vou colocar mais dois pontos aqui:
- Tem cliente que é muito escroto. Eu já trabalhei num local onde a gente tinha que comer em um local separado do pessoal "da casa", tinha catraca e tudo mais.
- Se tiver alguém da casa com o ego inflado, as decisões dele vão passar por cima de qualquer uma do terceirizado.

Há os pontos bons também, os salários geralmente são bons e término de contrato é uma coisa que sempre vai acontecer, então você acaba recebendo tudo o que tem direito e encontra outro emprego com uma certa facilidade.

 Sobre a "exploração", eu também concordo que isso é do profissional que se deixa ser explorado. Atualmente eu só faço hora extra se eu ver que a empresa tem uma boa cultura em relação à isso e se ficar claro qual é a forma de pagamento dessas horas extras.

**Comment by evansantos** [comment]
Quem pode ajudar nessa thread também é o @hjdesigner

**Comment by jeffersondanielss** [comment]
Ouvi esse podcast do Hipsters faz pouco tempo, achei que explicou bem como funciona esse mundo.
[Vida de Consultoria – Hipsters #97](https://hipsters.tech/vida-de-consultoria-hipsters-97/)

**Comment by andrearanha** [comment]
Existem sim os clientes escrotos, uma certa "inferioridade" por ser terceiro, etc.
Confesso que eu tive sorte, pois onde trabalho, fico muito feliz com o lado bom:
 - O RH não pode me encher o saco;
 - Faço um monte de horas extra, ganho dinheiro;
 - Salário bom por hora; horário flexível;
 - Meu trabalho é valorizado;
 - Minha gerência se esforça pra nos manter aqui, mesmo durante a crise.

**Comment by andremenezeswk** [comment]
Bom dia gente.

Achei essa discussão e queria compartilhar com vocês sobre como é trabalhar alocado em cliente na consultoria onde trabalho.

Temos um esforço para não errar em vários pontos que vocês citaram:

- Ambiente estressante
- Abandono no cliente
- Falta de perspectiva de mudanças

O esquema que o @rubenmarcus citou creio que seja a contratação por cooperativa.

Se quiserem dar uma lida rápida, tem um post no nosso site sobre o assunto (é possível ouvir o texto também):

https://wkrh.com.br/trabalhar-como-terceirizado-alocado-em-cliente/

Um abraço.
